shellhub/models/device.py

```diff
@@ -59,12 +60,12 @@ def __init__(self, api_object: shellhub.models.base.ShellHub, device_json):  # t
         self.info = ShellHubDeviceInfo(device_json["info"])
         self.public_key = device_json["public_key"]
         self.tenant_id = device_json["tenant_id"]
-        self.last_seen = device_json["last_seen"]
+        self.last_seen = datetime.fromisoformat(device_json["last_seen"])
         self.online = device_json["online"]
         self.namespace = device_json["namespace"]
         self.status = device_json["status"]
-        self.status_updated_at = device_json["status_updated_at"]
-        self.created_at = device_json["created_at"]
+        self.status_updated_at = datetime.now()     # We always want to track datetime when it is updated
```

Comment:
I disagree with this. I think it should always be the value returned from the API and not something we set ourselves.

This `__init__` is mainly a getter from the API return :)

Reply:
Yes, Right I must use datetime.fromisofromat(...) instead of datetime.now()

Reply:
Don't hesitate if you need more information on why